ui/admin/app/templates/scopes/scope/edit.hbs

```diff
@@ -13,55 +13,72 @@
   />
 {{/if}}
 
-<Rose::Layout::Page as |page|>
-
-  <page.breadcrumbs>
-    <Breadcrumbs::Container />
-  </page.breadcrumbs>
+<Rose::Layout::Page class='new-rose-page-layout' as |page|>
 
   <page.header>
-    {{#if @model.isGlobal}}
-      <Hds::Text::Display
-        @tag='h2'
-        @size='400'
-        @weight='bold'
-        class='header-spacing'
-      >
-        {{t 'resources.global.title'}}
-        <DocLink @doc='global' />
-      </Hds::Text::Display>
-      <Hds::Text::Body @tag='p' @size='300' class='p'>
-        {{t 'resources.global.description'}}
-      </Hds::Text::Body>
-    {{/if}}
-    {{#if @model.isOrg}}
-      <Hds::Text::Display
-        @tag='h2'
-        @size='400'
-        @weight='bold'
-        class='header-spacing'
-      >
-        {{t 'resources.org.title'}}
-        <DocLink @doc='org' />
-      </Hds::Text::Display>
-      <Hds::Text::Body @tag='p' @size='300' class='p'>
-        {{t 'resources.org.description'}}
-      </Hds::Text::Body>
-    {{/if}}
-    {{#if @model.isProject}}
-      <Hds::Text::Display
-        @tag='h2'
-        @size='400'
-        @weight='bold'
-        class='header-spacing'
-      >
-        {{t 'resources.project.title'}}
-        <DocLink @doc='project' />
-      </Hds::Text::Display>
-      <Hds::Text::Body @tag='p' @size='300' class='p'>
-        {{t 'resources.project.description'}}
-      </Hds::Text::Body>
-    {{/if}}
+    <Hds::PageHeader as |PH|>
+      <PH.Breadcrumb>
+        <Breadcrumbs::Container />
+      </PH.Breadcrumb>
+      {{#let
+        (array
+          (hash
+            key='isGlobal'
+            title='resources.global.title'
+            description='resources.global.description'
+            doc='global'
+          )
+          (hash
+            key='isOrg'
+            title='resources.org.title'
+            description='resources.org.description'
+            doc='org'
+          )
+          (hash
+            key='isProject'
+            title='resources.project.title'
+            description='resources.project.description'
+            doc='project'
+          )
+          (hash
+            key='isScope'
+            title='resources.scope.title'
+            description='resources.scope.description'
+            doc='scope'
+          )
+        )
+        as |scopes|
```

Comment:
could we move this to the controller? I think it would be easier to read and maintain in JS rather than building the structue directly in the template